docs/gettsim_objects/variables_out.md

```diff
@@ -28,6 +28,10 @@ You can find their individual calculation in the documentation of all {ref}`func
   - Monthly child benefit
 * - {func}`eink_st_tu <_gettsim.functions.eink_st_tu>`
   - Income Tax on tax unit level
+* - {func}`lohnst_m` <_gettsim.functions.lohnst_m>`
+  - Withholding tax
+* - {func}`soli_st_lohnst <_gettsim.functions.soli_st_lohnst>`
```

Comment:
Don't we want this function on a monthly basis? In that case, call it `soli_st_lohnst_m` and divide by 12 in the function body.

Reply:
is now on monthly basis